### PR TITLE
fix displayed rmrk to kusama

### LINKED ISSUE #4879: Network you are on
image
Rmrk should be renamed to Kusama. (in the navbar, when you are on the network/standart) 
image

### PR DESCRIPTION
**Pull request by maderi21**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4879 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [x] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Comment by maderi21** [comment]
This fix only solves the displayed chain in Navbar, however the URL is still set to /rmrk in `urlPrefix`. Do you want to change the url prefix as well?

File: components/navbar/ChainSelectDropdown.vue [hunk 1]
```diff
@@ -42,5 +42,13 @@ export default class ChainSelect extends Vue {
...
     this.$router.push({ path: `/${value}` })
   }
+
+  get chainName() {
+    if (this.selected === 'rmrk') {
+      return 'kusama'
+    } else {
+      return this.selected
+    }
+  }
```
Comment (Jarsen136) [inline components/navbar/ChainSelectDropdown.vue:52]: Let's use `getChainNameByPrefix` function in `utils/chain.ts`.

**Review by Jarsen136 — CHANGES_REQUESTED** [review]

File: components/navbar/ChainSelectDropdown.vue (hunk 1 above)
Comment (maderi21) [reply components/navbar/ChainSelectDropdown.vue:49]: done, thanks

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 38.37 USD/KSM ~ 0.782 $KSM
🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)
🔗 [0xaff653a54534e739a60d8072f48dcce1871778898fa1cac0ab1a07fa0fecbe5d](https://kusama.subscan.io/extrinsic/0xaff653a54534e739a60d8072f48dcce1871778898fa1cac0ab1a07fa0fecbe5d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 38.43 USD/KSM ~ 0.781 $KSM
🧗 [JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa](https://kusama.subscan.io/account/JHJGu2aWyAFBfcso8bR7ma82p15YAavyN6vQfJyWuhniNGa)
🔗 [0x04d6a5451175a7f67a24aca37f99d824bdc8aa39acdc9604f9eaedef155d922f](https://kusama.subscan.io/extrinsic/0x04d6a5451175a7f67a24aca37f99d824bdc8aa39acdc9604f9eaedef155d922f)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**